test/test_client_reset.cpp

```diff
@@ -231,25 +219,24 @@ TEST(ClientReset_InitialLocalChanges)
     ClientServerFixture fixture(dir, test_context);
     fixture.start();
 
-    Session session_1 = fixture.make_session(path_1, server_path);
+    DBRef db_1 = DB::create(make_client_replication(), path_1);
```

Comment:
This test predates core 6 and was doing something which is now quite weird (writing to a Realm via a second DB not linked to the sync session and not via a RealmCoordinator). It would have continued to work unchanged by continuing to use nonsync_transact_notify(), but since it isn't actually trying to test multiprocess things I made it normal instead.